packages/drafts-realm/leaflet-gtfs.gts

```diff
@@ -0,0 +1,256 @@
+import { CardDef } from 'https://cardstack.com/base/card-api';
+import NumberField from 'https://cardstack.com/base/number';
+import StringField from 'https://cardstack.com/base/string';
+import {
+  Component,
+  contains,
+  field,
+} from 'https://cardstack.com/base/card-api';
+import Modifier, { NamedArgs } from 'ember-modifier';
+//@ts-expect-error no types available
+import jszip from 'https://cdn.jsdelivr.net/npm/jszip@3.10.1/+esm';
+//@ts-expect-error no types available
+import Papa from 'https://cdn.jsdelivr.net/npm/papaparse@5.4.1/+esm';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { on } from '@ember/modifier';
+import { LeafletModifier } from './leaflet-map';
```

Comment:
I chose to build `LeafletGfts` off the modifier from the other file, but would it be more appropriate to extend the card definition? What’s the expected way for card authors to build on existing cards?

Reply:
I usually find myself starting with a new CardDef, too, but I think either option should be available. Extending the other class might make it easier if there are changes to the inherited fields, but on the flip side, it could also break things in your card. Good topic for discussion.